config/hwloc_internal.m4

```diff
@@ -93,6 +93,11 @@ AC_DEFUN([HWLOC_DEFINE_ARGS],[
                   AS_HELP_STRING([--disable-nvml],
                                  [Disable the NVML device discovery build (instead of only disabling NVML at runtime by default)]))
 
+    # RSMI?
+    AC_ARG_ENABLE([rsmi],
+                  AS_HELP_STRING([--disable-rsmi],
+                                 [Disable the RSMI device discovery]))
+
```

Comment:
Can you write something more clear such "ROCm SMI" in this documentation line?

Reply:
done